std_msgs/Header header

geometry_msgs/Vector3 amplitudes
geometry_msgs/Vector3 normalized_amplitudes
geometry_msgs/Vector3 phases